CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 /k-distribution/include/builtin/* @dwightguth
 /pending-documentation.md @dwightguth
+/Jenkinsfile @kframework/admin
```

Comment:
@dwightguth I don't know if this will work, because we need the rv/admin group don't we?